packages/angular-cli/tasks/build-webpack.ts

```diff
@@ -39,21 +36,18 @@ export default <any>Task.extend({
 
     return new Promise((resolve, reject) => {
       webpackCompiler.run((err: any, stats: any) => {
+        if (err) { reject(err); }
```

Comment:
I'm not sure it makes any difference. I can return here and on the final resolve/reject one though, what do you think?